core/src/test/scala/org/apache/spark/crypto/ShuffleEncryptionSuite.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.crypto
+
+import java.security.PrivilegedExceptionAction
+
+import org.apache.hadoop.security.{Credentials, UserGroupInformation}
+
+import org.apache.spark.{SparkConf, SparkFunSuite}
+import org.apache.spark.crypto.CryptoConf._
+import org.apache.spark.crypto.CryptoStreamUtils.SPARK_CHIMERA_CONF_PREFIX
+
+private[spark] class ShuffleEncryptionSuite extends SparkFunSuite {
+  val ugi = UserGroupInformation.createUserForTesting("testuser", Array("testgroup"))
+
+  test("Test Chimera configuration conversion") {
```

Comment:
This test is failing for me locally. I think at some point you changed the prefixes and now ithey're out of sync.

I suggest two things:
- getting rid of the `prefix` argument in `toChimeraConf`
- using string interpolation in the test

Here's my version of the test that is passing:

    test("Test Chimera configuration conversion") {
      val sparkKey1 = s"${SPARK_CHIMERA_CONF_PREFIX}a.b.c"
      val sparkVal1 = "val1"
      val chimeraKey1 = s"${CHIMERA_CONF_PREFIX}a.b.c"
  
      val sparkKey2 = s"${SPARK_CHIMERA_CONF_PREFIX.stripSuffix(".")}A.b.c"
      val sparkVal2 = "val2"
      val chimeraKey2 = s"${CHIMERA_CONF_PREFIX}A.b.c"
  
      val conf = new SparkConf()
      conf.set(sparkKey1, sparkVal1)
      conf.set(sparkKey2, sparkVal2)
  
      val props = toChimeraConf(conf)
      assert(props.getProperty(chimeraKey1) === sparkVal1)
      assert(!props.contains(chimeraKey2))
    }

While we're here, could you remove the redundant "Test" from the test description strings in this file?